src/sif/autocost.cc

```diff
@@ -330,7 +330,7 @@ AutoCost::AutoCost(const Costing costing, const Options& options)
   // Get the vehicle type - enter as string and convert to enum.
   // Used to set the surface factor - penalize some roads based on surface type.
   surface_factor_ = 0.5f;
-  std::string type = costing_options.transport_type();
+  const std::string& type = costing_options.transport_type();
```

Comment:
this change has nothing to do with the PR, but it blocked the 'lint-debug': https://circleci.com/gh/valhalla/valhalla/16358#tests/containers/0 , i think there may be a bug in clang-tide??